tests/include/igl/ray_mesh_intersect.cpp

```diff
@@ -30,6 +30,8 @@ TEST_CASE("ray_mesh_intersect: one_triangle", "[igl]")
   IGL_POP_FPE;
 }
 
+// https://github.com/libigl/libigl/issues/2363
+#ifndef _WIN32
```

Comment:
Rather than not compile the test on Windows, I would mark it as `[!mayfail]` so it is compiled but not run (and can easily be run manually). Something like this:

```c++
#ifdef _WIN32
#define IGL_MAYFAIL_WIN "[!mayfail]"
#else
#define IGL_MAYFAIL_WIN
#endif

TEST_CASE("ray_mesh_intersect: corner-case", "[igl]" IGL_MAYFAIL_WIN)
{
...
}
```

Reply:
oh nice! definitely let's do that.